lib/interface.bash

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
This looks like a good candidate for reuse in other Buildkite plugins that execute a Docker container.

Reply:
I pulled the plugin-specific stuff up to the top for that reason